hs_core/models.py

```diff
@@ -526,6 +526,9 @@ def update(cls, element_id, **kwargs):
             element.metadata.is_dirty = True
             element.metadata.save()
 
+        if(type(kwargs['abstract']) == list):
```

Comment:
```'abstract'``` -> ```'description'```
That should fix the failing unit test.

However, a broader question is why the value for this key is getting passed as a list. If we have to do this type check then there are many more elements where this check needs to be done.